test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -738,6 +738,28 @@ describe('S2S Adapter', function () {
         value: ['buzz']
       }]);
     });
+
+    it('adds s2sConfig adapterOptions to request for ORTB', () => {
```

Comment:
Could you replace the arrow function syntax with the normal `function () {` syntax?  This would better align with mocha's recommendations.